src/main/java/com/rocketden/main/model/problem/Problem.java

```diff
@@ -70,4 +70,8 @@ public void addProblemInput(ProblemInput problemInput) {
         problemInputs.add(problemInput);
         problemInput.setProblem(this);
     }
+
+    public void toggleApprovedStatus() {
+        approval ^= true;
+    }
```

Comment:
nit: I don't think this is used anywhere, so I think we can get rid of it

Reply:
k